### DISCUSSION TITLE
CarteACI_Mono_Autre_Routeur

### DISCUSSION THREAD

**Discussion by ArnaudNeo35** [FredM67/PVRouter-3-phase#80] [General]
Bonjour,

concernant cette instruction de câblage "CarteACI_Mono_Autre_Routeur.pdf" pour by-passer une ACI,
tous les commentaires ou vidéo disent qu'il n'est pas possible d'utiliser un routeur solaire avec un chauffe-eau disposant d'une carte ACI. Personne n'évoque cette astuce.
Existe-t-il des retours d'expérience, avec l'un de ces câblages ?

Dans mon cas, chauffe-eau Thermor Duralis 3kW 300L.

Merci

**Comment by FredM67** [comment]
Bonjour Arnaud,
Attention, il ne s'agit pas de by-passer la carte ACI !! La carte est toujours présente, fonctionnelle et assure à la fois la régulation en température et la coupure de sécurité.
Oui, énormément de personnes parlent pour ne rien dire, parlent sans savoir, pensent qu'ils ont la science infuse, .... C'est le propre de notre société actuelle, malheureusement !
De nombreuses personnes ont implémenté cette "astuce", je n'ai jamais eu aucun retour négatif. De toute façon, il n'y a de mystère dans ce câblage, il faut simplement mettre les différents composants (carte ACI, triac, ...) dans le bon ordre et le cas échéant rajouter un contacteur pour palier au fait que certains routeurs n'ont pas de circuit indépendant pour la charge (ce que je trouve, entre nous soit dit, déplorable, ... économie, économie !).
Fred
